packages/circuit-ui/components/SideNavigation/components/DesktopNavigation/DesktopNavigation.tsx

```diff
@@ -125,7 +131,7 @@ export function DesktopNavigation({
     activePrimaryLink && activePrimaryLink.secondaryGroups;
 
   return (
-    <Wrapper>
+    <Wrapper isLoading={Boolean(isLoading)}>
```

Comment:
Shouldn't `isLoading` already be a boolean?

Reply:
Oops meant to start a review, sorry, the rest is coming up

Reply:
It's a boolean or `undefined`, which is falsy, but that wasn't enough for TypeScript.